hoverfly/README.md

```diff
@@ -0,0 +1,37 @@
+# Hoverfly
+
+A simple container to run [hoverfly](https://docs.hoverfly.io/en/latest/). 
+
+TODO create a template
```

Comment:
@pcarney8 Is there a reason we're not including the template that we already have in this move? This actually resolves the issue that @makentenza is seeing with having to add `"-listen-on-host=0.0.0.0"` to the Dockerfile.

Reply:
templates are now in https://github.com/rht-labs/openshift-templates and I hate copy/pasting them everywhere.

Reply:
this is also whole PR is a literal copy/paste/remove from labs-ci-cd